third_party/rules_pycross/pycross/private/wheel_library.bzl

```diff
@@ -119,6 +129,34 @@ and py_test targets must specify either `legacy_create_init=False` or the global
 This option is required to support some packages which cannot handle the conversion to pkg-util style.
             """,
         ),
+        "patch_args": attr.string_list(
+            default = ["-p0"],
+            doc =
+                "The arguments given to the patch tool. Defaults to -p0, " +
+                "however -p1 will usually be needed for patches generated by " +
+                "git. If multiple -p arguments are specified, the last one will take effect.",
+        ),
+        "patch_dir": attr.string(
+            default = "",
+            doc = "The directory from which to invoke the patch_tool.",
+        ),
+        "patch_tool": attr.string(
+            doc = "The patch(1) utility from the host to use. " +
+                  "If set, overrides `patch_tool_target`.",
+        ),
+        "patch_tool_target": attr.label(
+            executable = True,
+            cfg = "exec",
+            doc = "The label of the patch(1) utility to use. " +
+                  "Only used if `patch_tool` is not set.",
```

Comment:
Do we restrict users to only use `patch(1)` tool?

Reply:
No. This is simply copied from the `http_archive` implementation.  I'm happy to change the wording, but keeping it similar to `http_archive` seemed reasonable at the time at least. I think part of the reason for writing it as `patch(1)` explicitly is to imply the API (i.e. patch on stdin).
WDYT?

https://github.com/bazelbuild/bazel/blob/0226327c0ac03d818c211a718494c7c7cdc9bafc/tools/build_defs/repo/http.bzl#L327-L330